apps/build.gradle

```diff
@@ -2,14 +2,14 @@
 
 plugins {
     id "java"
-    id "com.github.node-gradle.node" version "3.6.0"
+    id "com.github.node-gradle.node" version "7.0.1"
 }
 
 def nodeSpec = {
     version = '19.9.0'
     yarnVersion = '1.22.19'
     npmVersion = "9.5.1"
-    download = true
+    download = System.getenv('NODE_SPEC_DOWNLOAD') ? System.getenv('NODE_SPEC_DOWNLOAD').toBoolean(): true
```

Comment:
What do you think of checking the 'CI' environment variable
see https://circleci.com/docs/variables/#built-in-environment-variables

Reply:
o nice, a now i see its default set by circle ci